### ISSUE TITLE
brain-view2 Broken?

### ISSUE THREAD

**Issue by gdevenyi** [CobraLab/MINC-VM#26]
User reports
```
 'brain-view2: error while loading shared libraries: libbicpl.so.2.0.1: cannot open shared object file: No such file or directory'
```

minc* commands work.

**Comment by gdevenyi** [comment]
Looks like the brain-view2 build system is no braindead it can't find libraries it was built against.

Solution is to add minc-toolkit's lib directory to LD_LIBRARY_PATH

**Comment by bcdarwin** [comment]
A collaborator has also reported this is the case for Pydpiper pipelines, which I suspect is Pyminc also requiring the LD_LIBRARY_PATH set (since it always looks for libraries dynamically).

**Comment by gdevenyi** [comment]
Latest version has this solved.

**Comment by gdevenyi** [comment]
Fixed by 599a39a336010ee09a0390fa06204480f16c9bd6
